explainer.md

```diff
@@ -143,17 +143,19 @@ MyScript also provides helper classes and SDKs to manage stroke capture. Applica
 Handwriting recognizers on different platforms have different features. Web applications can query their feature support and decide if the API is suitable for their use case.
 
 ```JavaScript
-await navigator.queryHandwritingRecognizerSupport('writingArea')
-// => true
-
-await navigator.queryHandwritingRecognizerSupport('alternatives')
-// => true
-
-await navigator.queryHandwritingRecognizerSupport('supportedLanguages')
-// => ['en', 'zh']
-
-await navigator.queryHandwritingRecognizerSupport('supportedTypes')
-// => ['text', 'number']
+// The list of features to detect.
+await navigator.queryHandwritingRecognizerSupport([
```

Comment:
This should probably be a dictionary rather than a list. If needed, keys could have values too.

Reply:
In this case, what would the return value be?

`true / false` to indicate if the feature value specified in the arguments are supported?